README.md

```diff
@@ -20,9 +22,33 @@ Supercell Wx supports the following 64-bit operating systems:
 - Windows 10 (1809 or later)
 - Windows 11
 - Linux
-  - EndeavourOS
+  - Arch Linux (EndeavourOS, SteamOS [Steam Deck], and other Arch derivatives)
   - Fedora Linux 34+
   - openSUSE Tumbleweed
-  - SteamOS (Steam Deck)
   - Ubuntu 22.04+
   - Most distributions supporting the GCC Standard C++ Library 11+
+ 
+## Linux Dependencies
+
+Supercell Wx requires the following Linux dependencies:
+
+- Linux/X11 (Wayland works fine too) with support for GCC 11 and OpenGL
+- X11/XCB libraries including xcb-cursor
+ 
+## FAQ
+
+Frequently asked questions:
+
+- Q: Why is the map black when loading for the first time?
+  
+  - A. You must obtain a free API key from either (or both) [MapTiler](https://cloud.maptiler.com/auth/widget?next=https://cloud.maptiler.com/maps/) which reuqires no credit/debit card, or [Mapbox](https://account.mapbox.com/) which does require a credit/debit card, but as of writing, you will get 200K free requests per month, which you will never go over by yourself.
```

Comment:
reuqires -> requires

Recommend rewording "which you will never go over by yourself" to "which should be sufficient for an individual user"

Reply:
Ah! Good catch. Fixed the typo and re-worded the sentence for quality.